cabby/geo/walk.py

```diff
@@ -322,7 +319,7 @@ def get_landmark_if_tag_exists(self,
     return None
 
 
-  def pick_prominent_pivot(self, df_pivots: GeoDataFrame, end_point: Dict
+  def pick_prominent_pivot(self, df_pivots: GeoDataFrame, end_point: Dict, small: bool = False
   ) -> Optional[GeoSeries]:
```

Comment:
Please break this up over multiple lines. Also, please provided types for the keys and values for the Dict.